content/posts/irods-development-update-july-2022.md

```diff
@@ -0,0 +1,23 @@
+Title: iRODS Development Update: July 2022
+Date: 2022-07-29 17:00
+Author: Kory Draughn
+Slug: irods-development-update-july-2022
+Status: published
+
+
+July was an exciting month. The [iRODS User Group Meeting](https://irods.org/ugm2022) happened and best of all, it was in person!
+
+It was great seeing everyone and hearing about the interesting things they've been working on using iRODS. We're reviewing footage and hope to post videos very soon.
+
+Our interns are working hard to put the finishing touches on their summer work. They'll be talking about their experience at [TRiRODS](https://irods.org/trirods) on August 10th. We hope you'll listen in.
+
+Aside from the work being done by the interns, we've started working on improving the documentation. This documentation will cover basic and advanced concepts in iRODS. You can expect to find information on topics such as data objects, logical locking, and high-level operations. We're also working on adding an experimental section called **Policy Cookbook**. Here you'll find various policy-related topics covering things such as metadata-based synchronization, simulating user quotas, and more. You can track the progress of this work at the following locations:
+- https://github.com/irods/irods_docs/pull/149
+- https://github.com/irods/irods_docs/pull/148
```

Comment:
@trel Will these render correctly?

Reply:
add newline to separate from 'locations:'

Reply:
or just say it's coming soon.... and no links.  since they'll not be relevant over time.

Reply:
Ok. I'll do that.